docs/static/scripts/fetch-templates.js

```diff
@@ -113,6 +113,11 @@ async function getTemplateRuntime(template){
 
 }
 
+function cleanupUndefined(data) {
```

Comment:
Does this really warrant its own function rather than just being a one-liner in fetchTemplates() before writeFileSync gets called?